src/std/foreign.ss

```diff
@@ -52,11 +52,16 @@
       ;;    If no cleanup function is provided, a c function is created <struct-name>_ffi_free
       ;;    this function frees the struct pointer as well as any string members if
       ;;    they were set.
-      (define-macro (define-c-struct struct #!optional (members '()) release-function)
+      ;; compatible-tags => list of symbols representing additional C type declarations compatible with struct
+      ;;    This is usually the name of a typedef alias, when it differs from the C struct tag.
```

Comment:
The documentation isn't clear, and there aren't examples in .md files either.

Could you clarify what that does, and maybe provide examples in the according .md file?

Reply:
Aha, so it's a Gambit thing, rather than a C thing. You should refer to this paragraph (by name rather than by number, or both), in your comment and/or in the documentation.